tests/functional/stack-parameters-overwrite-action/source/data/config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "action": "testConnection",
+  "image_parameters": {
+    "global_config": {
+      "db": {
+        "authType": "kerberos",
+        "kerberos": {
+          "principal": "%env(string:HIVE_DB_KERBEROS_PRINCIPAL)%",
+          "config": "%env(string:HIVE_DB_KERBEROS_KRB5_CONF)%",
+          "#keytab": "%env(string:HIVE_DB_KERBEROS_KEYTAB_ENCODED)%"
+        },
+        "ssl": {
+          "enabled": true,
+          "ca": "%env(string:HIVE_DB_KERBEROS_SSL_CERT_JKS)%",
+          "caFileType": "jks"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "db": {
+      "host": "%env(string:HIVE_DB_KERBEROS_HOST)%",
+      "port":  "%env(int:HIVE_DB_KERBEROS_PORT)%",
+      "database": "%env(string:HIVE_DB_KERBEROS_DATABASE)%",
+      "authType": "BAD",
```

Comment:
Hodnoty v `parameters` maju mensiu prioritu.

Reply:
neměl by právě mít uživatel možnost přetížit globální nastavení? takhle když si do konfigurace nahrajou jiný certifikát tak jim to vždycky přepíšeš a nemusí jim to potom fungovat

Reply:
asi ukaze az prax co je lepsie, ... ale mozem to v tejto prvej verzii dat aj naopak.

Reply:
z logiky mi to přijde lepší opačně a pokud to tak není nutně mít tak bych to otočil

Reply:
ja som vychadzal z toho, ze sa to momentalne nebude dat v UI menit, bude to iba readonly, ... ale mas pravdu:
vymeneneL https://github.com/keboola/db-extractor-hive/pull/15/commits/cad10bf181fcfd83a2e44775266b36300a70a73b